README.md

```diff
@@ -1 +1,58 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is my first project in microverse.
+
+## Built With
+
+- HTML,CSS
+- Github
+- Linters
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] [OPTIONAL] As the previous reviewer did not mention this, I kindly suggest you add a live demo link or remove this section entirely.